python/ray/util/client/server/server.py

```diff
@@ -70,45 +70,18 @@ def Init(self, request: ray_client_pb2.InitRequest,
             job_config.client_job = True
         else:
             job_config = None
-        current_job_config = None
         with disable_client_hook():
-            if ray.is_initialized():
-                worker = ray.worker.global_worker
-                current_job_config = worker.core_worker.get_job_config()
```

Comment:
Could you leave this is initialized check in and log a warning if this happens? 


(Don't raise an error because this codepath might be hit in tests still)

Reply:
Put this code back because it caused a test to fail (not sure if it's actually used)